public/app/features/plugins/admin/pages/PluginDetails.tsx

```diff
@@ -51,19 +51,27 @@ export default function PluginDetails({ match }: PluginDetailsProps): JSX.Elemen
             <PluginLogo
               src={plugin.info.logos.small}
               className={css`
-                object-fit: cover;
+                object-fit: contain;
                 width: 100%;
                 height: 68px;
                 max-width: 68px;
               `}
             />
 
             <div className={styles.headerWrapper}>
-              <h1>{plugin.name}</h1>
+              <h1 className={styles.header}>
```

Comment:
Hey Jack 😄 , because we are moving towards more a11y pages and components, I took the liberty to check this PR with that lense, these are suggestions to make it more a11y :) (_feel free to adjust it if you consider this is the right time to apply a11y into this component_)

* It will be better to use a `nav` tag to make it more semantic 
* We can change as well the name of the class
* And lastly, even though this is a simple breadcrumb with only 1 level, maybe a good idea is wrapped into a list and add the `/` via CSS, this makes a difference for screen readers

```
              <nav aria-label="Breadcrumb" className={styles.header}>

```


For inspiration, this resource can help :). https://www.w3.org/TR/wai-aria-practices-1.1/examples/breadcrumb/index.htm